README.md

```diff
@@ -1,110 +1 @@
-# NameHash NameGuard
-
-![Tests](https://github.com/namehash/nameguard/actions/workflows/ci_api.yml/badge.svg?branch=main)
-![Coverage](https://raw.githubusercontent.com/namehash/nameguard/main/coverage_badge.svg)
```

Comment:
@djstrong @Carbon225 @notrab Appreciate your help to confirm what we should do with this coverage badge.

One of the confusing things about this badge for me is: Exactly what within this monorepo is this coverage related to? There's so many different sub-projects.

I see that the last time this badge was automatically updated was 4 months ago by a GitHub Action: https://github.com/namehash/nameguard/commits/dd6f9af2587348a4808cf15356338c10181d00ec/coverage_badge.svg

Is my memory on this correct? Did we temporarily add a GitHub Action for this badge and then later remove the GitHub Action but never cleaned up this badge properly?

I still see a GitHub Action that generates a coverage badge, but perhaps this is only for the `/api` project? https://github.com/namehash/nameguard/blob/dd6f9af2587348a4808cf15356338c10181d00ec/.github/workflows/ci_api.yml#L82-L84

This is confusing because when I check https://github.com/namehash/nameguard/tree/main/api I don't see any coverage badge there.

Is the solution to remove the coverage badge from the root of our monorepo and move it into /api with a reference to it inside /api/README.md? This might involve some updates to the related GitHub action?

Appreciate your advice. Thank you.

Reply:
This badge was related to the API, so this should really be placed inside the `api` folder and sourced from the README accordingly.

Reply:
Yes, it still working and relates to Python API. In https://github.com/namehash/nameguard/pull/261 I moved the badge to `api` directory.
It was updated 4 months ago because the coverage has not changed since then.

Probably there is problem with the first badge because now API tests are not run every time.